apps/system/style/statusbar/statusbar.css

```diff
@@ -6,7 +6,8 @@
   height: 2rem;
 }
 
-#screen.rocketbar-focused .gesture-panel {
+#screen.rocketbar-focused #left-panel,
```

Comment:
Can you add a quick comment here? Something about that this is for allowing the utility tray to open, but not edge gestures?